bitnami/etcd/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/etcd
   - https://coreos.com/etcd/
-version: 8.8.2
+version: 8.8.3
```

Comment:
The change you propose is adding new variables, therefore it is not fixing a simple bug. It would be more appropriate, following the semver.org guidelines, for the version to increase in a minor and not a patch.
Therefore, the value should be 8.9.0.

Reply:
@corico44 great catch -- fixed!